data/js/about-home.js

```diff
@@ -8,27 +8,25 @@ var searchIconAndTextContainer = document.querySelector('#searchIconAndTextConta
  * @param {string} snippetContent - The content to inject
  */
 function showSnippet(snippetContent) {
-    var button;
     var addonSnippet = document.querySelector('#allaboard_snippet');
```

Comment:
This is a nit and feel free to ignore, but `getElementById` is much faster than `querySelector` if all you're getting is an single ID.